theme_buzzy/views/new_page_template.xml

```diff
@@ -87,16 +87,13 @@
 </template>
```

Comment:
Commit message:
- the commit message should start with a verb :wink: => adapt s_timeline
- nitpick: space between PR et [1] :slightly_smiling_face: